Tests/Functional/Controller/ImportTest.php

```diff
@@ -7,185 +7,232 @@
 
 namespace ShopwarePlugins\Connect\Tests\Functional\Controller;
 
-use ShopwarePlugins\Connect\Tests\DatabaseTestCaseTrait;
+use ShopwarePlugins\Connect\Tests\TestClient;
+use ShopwarePlugins\Connect\Tests\WebTestCaseTrait;
+use Symfony\Component\HttpFoundation\Response;
 
-class ImportTest extends \Enlight_Components_Test_Plugin_TestCase
+class ImportTest extends \PHPUnit_Framework_TestCase
 {
-    use DatabaseTestCaseTrait;
+    use WebTestCaseTrait;
```

Comment:
does the webTestCaseTrait roll back changes to the db ->
because if you invoke the backend controllers something could be changed in the db 
even if you do no db-queries in the testCase

Reply:
Yes the KernelTestCase already wraps everything into a transaction and the WebTestCase is just an extension